client/test/extension.test.ts

```diff
@@ -1,22 +1,22 @@
-//
-// Note: This example test is leveraging the Mocha test framework.
-// Please refer to their documentation on https://mochajs.org/ for help.
-//
+// //
+// // Note: This example test is leveraging the Mocha test framework.
+// // Please refer to their documentation on https://mochajs.org/ for help.
```

Comment:
The tests were throwing compilation errors, which is why I commented them out.  Can revert